threejs/src/index.js

```diff
@@ -43,6 +45,26 @@ export default class ZestyBanner extends Mesh {
       }
     });
     this.onClick = this.onClick.bind(this);
+
+    setInterval(() => {
+      const camera = this.getCamera();
```

Comment:
Feel this entire section could be broken out to a fn, something like `checkIfBeingViewedForRefresh` so it's clear to understand what's happening here:

```javascript
setInterval(() => {
  checkIfBeingViewedForRefresh(this);
}, AD_REFRESH_INTERVAL);
```

WDYT?

Reply:
Yeah that sounds good, I'll split it out